.postcssrc.yml

```diff
@@ -1,3 +1 @@
 plugins:
```

Comment:
If we are removing postcss could we remove this file completely?

Reply:
For some reason it needs it. There's a bunch of other dependencies that apparently need this file.